s2core/src/main/scala/com/kakao/s2graph/core/storage/hbase/AsynchbaseStorage.scala

```diff
@@ -79,6 +79,22 @@ class AsynchbaseStorage(override val config: Config, vertexCache: Cache[Integer,
   val FailProb = config.getDouble("hbase.fail.prob")
   val LockExpireDuration = config.getInt("lock.expire.time")
 
+  private def initClients(flushInterval: Int): Map[String, HBaseClient] = {
+    Service.findAllConn().distinct.map { zkQuorum =>
+      zkQuorum -> AsynchbaseStorage.makeClient(config,
+        "hbase.zookeeper.quorum" -> zkQuorum,"hbase.rpcs.buffered_flush_interval" -> flushInterval.toString)
+    } toMap
+  }
+  val clientsWithInterval = initClients(clientFlushInterval)
+  val clientsWithZeroInterval = initClients(0.toShort)
+
+  def client(zkQuorum: String, withWait: Boolean = false): HBaseClient = {
+    val pool = if (withWait) clientsWithZeroInterval else clientsWithInterval
+    pool.get(zkQuorum).getOrElse(
+      throw new RuntimeException(s"HBaseClient for ${zkQuorum}, ${withWait} is not initialized.")
```

Comment:
I am not sure what we supposed to do on this case. should be create client? then I think create client is blocking call, and should be done asynchronously(like Model update). what do you guys think? do we need this support?